### PR TITLE
Update messages.json RUS

### PR DESCRIPTION
**Pull request by wvxwxvw**

### CODE REVIEW

File: src/_locales/ru/messages.json [hunk 1]
```diff
@@ -1,214 +1,214 @@
...
+    "message": "Может обмениваться сообщениями с внешними приложениями на ПК",
     "description": "Description of nativeMessaging Text"
   },
   "telemetryText": {
     "message": "Только Firefox",
     "description": "Description of telemetry Text"
   },
   "themeText": {
-    "message": "Можно обновить тему браузера",
+    "message": "Может получать, обновлять и удалять обновленя тем браузера",
```
Comment (tsl143) [inline src/_locales/ru/messages.json:83]:
```suggestion
    "message": "Может изменять внешний вид браузера",
```

File: src/_locales/ru/messages.json [hunk 2]
```diff
@@ -1,214 +1,214 @@
...
     "description": "Description of theme Text"
   },
   "cprT": {
-    "message": "Можно скопировать данные с веб-страницы",
+    "message": "Может копировать данные с веб-страницы",
     "description": "Description of clipboardRead Text "
   },
   "idleText": {
-    "message": "Может узнать, когда система пользователя находится в режиме ожидания, заблокирована или активна",
+    "message": "Может узнавать статус ОС: режим ожидания, блокировка, неактивность",
```
Comment (tsl143) [inline src/_locales/ru/messages.json:91]:
```suggestion
    "message": "Может определить статус ОС: режим ожидания, блокировка, неактивность",
```

File: src/_locales/ru/messages.json [hunk 3]
```diff
@@ -1,214 +1,214 @@
...
     "description": "Description of browserSettings Text"
   },
   "cookiesText": {
-    "message": "Может получать и устанавливать файлы cookie, а также получать уведомления об их изменении",
+    "message": "Может получать и устанавливать файлы cookie, а также знать об их изменении",
     "description": "Description of cookies Text"
   },
   "downloadsText": {
-    "message": "Может загружать файлы, отменить, приостановить, возобновить загрузку",
+    "message": "Может загружать файлы, отменять, приостановливать, возобновлять загрузку",
```
Comment (tsl143) [inline src/_locales/ru/messages.json:143]:
```suggestion
    "message": "Может загружать файлы, отменять, приостанавливать, возобновлять загрузку",
```

File: src/_locales/ru/messages.json [hunk 4]
```diff
@@ -1,214 +1,214 @@
...
     "description": "Description of find Text"
   },
   "historyText": {
-    "message": "Может взаимодействовать и манипулировать историей браузера",
+    "message": "Может взаимодействовать и управлять историей браузера",
     "description": "Description of history Text"
   },
   "mocT": {
-    "message": "Только Firefox",
+    "message": "Может переопределать или создавать свое контекстное меню",
```
Comment (tsl143) [inline src/_locales/ru/messages.json:111]:
```suggestion
    "message": "Может переопределять или создавать своё контекстное меню",
```

**Comment by tsl143** [comment]
@wvxwxvw  Thank you so much for the translations

**Comment by championshuttler** [comment]
@wvxwxvw Thank you so much 💯 .

New version is released with your translations https://addons.mozilla.org/firefox/addon/permission-inspector/

**Comment by wvxwxvw** [comment]
Thanks!

**Pull request merged**